src/CodeCoverage.php

```diff
@@ -909,4 +839,26 @@ private function selectDriver()
             return new PHP_CodeCoverage_Driver_Xdebug;
         }
     }
+
+    /**
+     * Initialise the data with the executable/dead lines from each file in the whitelist
+     */
+    private function initData()
+    {
+        foreach ($this->filter()->getWhitelist() as $file) {
+            $this->driver->start();
+            include_once($file);
```

Comment:
I think we should try to make the `include_once` optional. Could we resurrect the `processUncoveredFileFromWhitelist` flag and only do the `include_once` when the flag is `true`?